src/main/java/models/User.java

```diff
@@ -0,0 +1,77 @@
+package models;
+
+import java.util.Objects;
+
+public class User {
+    private String name;
+    private String nickName;
+    private Integer id;
+    private Integer gameCount = 0;
+
+    public User(String name, String nickName, Integer id) {
+        this.name = name;
+        this.nickName = nickName;
+        this.id = id;
+    }
+
+    public User(String name, String nickName) {
+        this.name = name;
+        this.nickName = nickName;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getNickName() {
+        return nickName;
+    }
+
+    public void setNickName(String nickName) {
+        this.nickName = nickName;
+    }
+
+    public Integer getId() {
+        return id;
+    }
+
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    public Integer getGameCount() {
+        return gameCount;
+    }
+
+    public void setGameCount(Integer gameCount) {
+        this.gameCount = gameCount;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.

Reply:
- [x] added braces